openshift-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/build/OpenshiftBuildMojo.java

```diff
@@ -44,7 +44,7 @@
      * docker mode.
      */
     @Parameter(property = "jkube.build.strategy")
-    protected OpenShiftBuildStrategy buildStrategy = OpenShiftBuildStrategy.s2i;
+    protected JKubeBuildStrategy buildStrategy = JKubeBuildStrategy.s2i;
```

Comment:
This parameter belongs now to BuildMojo (jib build strategy is applicable both of OpenShift and Kubernetes)

Reply:
Ok, just saw it's in AbstractDockerMojo, please remove from here.